src/Sort.c

```diff
@@ -110,13 +118,17 @@ int mergeSort(int arr[], int low, int high){
 return 0;
 }
 
+// QUICK SORT OF INT ARRAY
+
 // dividing array into two partitions
 int partition(int arr[], int low, int high){
   // last Element becomes pivot
   int pivot = arr[high];
```

Comment:
might have to clean up comments within function bodies and move the explanation on top of the function head when we have doxygen parse the code to generate docs